packages/connectivity_plus/README.md

```diff
@@ -11,8 +11,8 @@ themselves accordingly. It can distinguish between cellular vs WiFi connection.
 
 ## Platform Support
 
-| Android | iOS | MacOS | Web | Linux | Window |
-| :-----: | :-: | :---: | :-: | :---: | :----: |
+| Android | iOS | MacOS | Web | Linux | Windows |
+| :-----: | :-: | :---: | :-: | :---: |  :----: |
```

Comment:
Small nit-pick, the space in the extra space in the `| _:----: |` should be an extra dash instead: `| :-----: |`

Reply:
@miquelbeltran done! Thanks.